### PR TITLE
feature/19285 - Copy changes

### PR DESCRIPTION
**Pull request by pris54**
Copy changes. Feature toggling was already implemented for some of the required changes.

### CODE REVIEW

File: admin/views/school/pupil-register.ejs [hunk 1]
```diff
@@ -16,20 +16,28 @@
...
+                -->
             </p>
             <div class="button-wrapper">
                 <% if (isFeatureEnabled('pupilAdd')) { %>
                 <a class="button button-secondary" href="/school/pupil/add"> Add pupil </a>
                 <% } else { %>
                 <a class="button button-secondary disabled-link" href="#"> Add pupil </a>
                 <% } %>
-
+                <!--
```
Comment (jon-shipley) [inline admin/views/school/pupil-register.ejs]: Does this need to be commented out?  Does't the feature toggling do enough?

File: admin/views/school/pupil-register.ejs [hunk 2]
```diff
@@ -52,9 +60,12 @@
                     <td<% if (highlight && highlight.has(p.urlSlug.toString())) { %> class="highlight-item"<% } %>>
                         <div class="highlight-wrapper">
                             <% if (isFeatureEnabled('pupilEdit')) { %>
+                                <span class="bold-small truncate"><%= `${p.lastName}, ${p.foreName}` %></span>
+                                <!--
```
Comment (jon-shipley) [inline admin/views/school/pupil-register.ejs]: Does this need to be commented out?  Does the feature toggling not do enough?

File: admin/views/school/pupil-register.ejs [hunk 3]
```diff
@@ -16,20 +16,28 @@
         <div>
             <h1 class="heading-xlarge">Pupil register</h1>
             <p class="lede">
+                During the trial pupils will take the check anonymously. Below is a list of 36 pre-populated pupil records.
+                <ul>
+                    <li><a href="https://irp0dn.axshare.com/prototype/login/irp0dn/#PupilReg=default&CSUM=1">As per Prototype</a></li>
```
Comment (jon-shipley) [inline admin/views/school/pupil-register.ejs]: The ampersand in the href attribute should be encoded: `&amp;`

File: admin/views/school/pupil-register.ejs (hunk 1 above)
Comment (pris54) [reply admin/views/school/pupil-register.ejs]: Comments can be removed now that it's been confirmed that feature toggling will work on this environment. I'll do the necessary changes.

File: admin/views/school/pupil-register.ejs (hunk 2 above)
Comment (pris54) [reply admin/views/school/pupil-register.ejs]: Same as above, feature toggling is enough.

**Pull request merged**